backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/edges/AbstractUsageNestedUsageEdgeDescriptionProvider.java

```diff
@@ -98,4 +103,29 @@ private EdgeStyle createEdgeStyle() {
                 .targetArrowStyle(ArrowStyle.NONE)
                 .build();
     }
+
+    private SourceEdgeEndReconnectionTool createSourceReconnectTool() {
```

Comment:
May be this method could be moved into `AbstractEdgeDescriptionProvider`? What do you think about it?

Reply:
Agree. Done.